Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -334,7 +334,11 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
             if not isinstance(assignee, User):
                 raise TypeError("Must assign to '{}', not '{}'".format(User, type(assignee)))
             issue._assignee = self.user(name=assignee.name, username=assignee.username, email=assignee.email)
-            update_dict['assignees'] = [issue._assignee.username]
+            assignees = [issue._assignee.username]
+            if not self.add_assignees(issue, assignees):
+                sys.stderr.write('Could not add assignee(s) to issue')
```

Comment:
Seems like `add_assignees` should own the error here.

Reply:
Makes sense.